### ISSUE TITLE
Use less hacky way of getting around credential manager issue

### ISSUE THREAD

**Issue by rylev** [rust-lang/rustwide#4]
Currently, when cloning repos, rustwide passes `ghost:ghost@` at the beginning of the url's host to GitHub. This informs GitHub and the underlying credential manager on the host to use these credentials avoiding any prompts. Ultimately, these credentials are ignored because we are only cloning public repos. 

This is a bit hacky. Another possible solution is [writing our own credential manager](https://git-scm.com/docs/api-credentials) that "does the right thing". We can then [pass an argument to git clone](https://stackoverflow.com/questions/13198143/how-do-i-disable-gits-credential-helper-for-a-single-repository/44033298#44033298) to use this credential manager.

**Comment by ecstatic-morse** [comment]
To expand on the "write your own credential manager" idea, we could add the following section to a config file to duplicate the existing behavior. The question is whether this would go into the user's config (`~/.gitconfig` on Linux), or into `.git/config` in crater's source dir. AFAICT, `.git/config` doesn't actually get used when running `git clone`, even if you're in a git repo. Honestly, hard-coding the dummy credentials into the `clone` invocation seems less error prone.

```
[credential]
	helper = "!f() { echo username=ghost; echo password=ghost; }; f"
```

**Comment by emilyalbini** [comment]
Fixed in rustwide 0.2.0.
